turbo.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "globalEnv": ["GITHUB_TOKEN"],
+  "tasks": {
+    "build": {
+      "dependsOn": ["^build"]
+    },
+    "format": {
+      "dependsOn": ["^format"]
+    },
+    "lint": {
+      "inputs": ["**/*.ts", "**/*.tsx", "**/*.js", "**/*.css", "**/biome.json"],
+      "dependsOn": ["build"]
+    },
+    "size-limit": {
+      "dependsOn": ["build"]
+    },
+    "test": {
+      "dependsOn": ["build"]
```

Comment:
This is something cool—here’s where we depend on another task, so it will get fancy here and **only run the minimal build commands needed.** So now, you don’t need to remember to run `pnpm run build && pnpm test`; you can just let Turborepo handle it for you.